test/models/driver_test.rb

```diff
@@ -60,22 +60,68 @@
 
   # Tests for methods you create should go here
   describe "custom methods" do
+    before do
+      @new_driver = Driver.create(name: "Mike", vin: "123", available: true)
+      @new_passenger = Passenger.create(name: "Mike", phone_num: "111-111-1211")
+
+    end
+    let(:trip_1) {
+      Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: Date.today, rating: 5, cost: 1000)
+    }
+    let(:trip_2) {
+      Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: Date.today, rating: 3, cost: 6334)
+    }
+    let(:trip_3){
+      Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: Date.today, rating: 3, cost: 1234)
+    }
     describe "average rating" do
-      # Your code here
+      it "returns average rating" do
+        trip_1
+        trip_2
+        trip_3
+        expect(@new_driver.average_rating).must_be_close_to 3.7
+      end
+
+      it "returns 0 if there is no trip" do
+        expect(@new_driver.average_rating).must_equal 0
     end
+  end
```

Comment:
Nitpick: Your indentation is off here-- this ends up being pretty impactful, because I was totally lost and confused about why your tests were behaving the way they were, since your indentation was off. Watch out!!!!

Note that it's really difficult to see that the `before` block applies to the tests below because of the indentation.